pkg/skaffold/deploy/util.go

```diff
@@ -133,6 +136,23 @@ func dumpToFile(renderedManifests string, filepath string) error {
 	return err
 }
 
+// Returns relative path pointing to the GCS temp dir
+func downloadManifestsFromGCS(manifests []string) (string, error) {
+	if err := os.MkdirAll(manifestTmpDir, os.ModePerm); err != nil {
+		return "", fmt.Errorf("failed to create the tmp directory: %w", err)
+	}
+	for _, manifest := range manifests {
+		if manifest == "" || !strings.HasPrefix(manifest, gcsPrefix) {
+			return "", fmt.Errorf("%v is not a valid GCS path", manifest)
```

Comment:
please cleanup the manifestTmpDir here using `defer os.RemoveAll(manifestTmpDir)`

Reply:
See the comment above.